src/components/TodoItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+function TodoItem(props) {
+    return (
+        <li className="">
+            <div className="view">
+                <input
+                    className="toggle"
+                    type="checkbox"
+                    checked={props.completed}
+                    onChange={props.onChange}
+                    onClick={(e) => {
+                        e.nativeEvent.target.parentNode.classList.toggle('completed')

```

Comment:
You shouldn't use `DOM` methods in react. Remove this `onClick` and add conditional rendering of classname to needed node